supervisor/supervisor.mk

```diff
@@ -119,13 +124,20 @@ ifndef USB_INTERFACE_NAME
 USB_INTERFACE_NAME = "CircuitPython"
 endif
 
+ifndef USB_WEBUSB_URL
+USB_WEBUSB_URL = "www.circuitpython.org"
```

Comment:
`www.circuitpython.org` redirects to `circuitpython.org`, so leave off the `www.`. But if this is an actual URL, shouldn't it have `https://`?

Reply:
No objection to leaving off the www. The C struct that uses this has a different field that controls the prefix, look for bScheme.

Reply:
I just updated the supervisor.mk file.